index.md

```diff
@@ -1,11 +1,10 @@
 ---
 ---
+
 # Summary Reports
-- [2025-12-13 00:33:04 (0907c245a4eebbeb44daa2626c11f23461e89367)](run.20251213003304.0907c245a4eebbeb44daa2626c11f23461e89367.html)
-- [2025-12-13 00:17:59 (b3c55d5a8c91646298d9a4b5a292f66cf66130e2)](run.20251213001759.b3c55d5a8c91646298d9a4b5a292f66cf66130e2.html)
-- [2025-12-12 23:48:00 (199a5c8d028532bb6bb63237de33860c7272b508)](run.20251212234800.199a5c8d028532bb6bb63237de33860c7272b508.html)
-- [2025-12-12 22:43:56 (02876960f2e095bb8a41e0d183f72fd6396d2bae)](run.20251212224356.02876960f2e095bb8a41e0d183f72fd6396d2bae.html)
-- [2025-12-12 22:19:33 (f2763720ea0e262ebd57ee0c881c3abd4660f39c)](run.20251212221933.f2763720ea0e262ebd57ee0c881c3abd4660f39c.html)
-- [2025-12-12 22:04:48 (f2763720ea0e262ebd57ee0c881c3abd4660f39c)](run.20251212220448.f2763720ea0e262ebd57ee0c881c3abd4660f39c.html)
-- [2025-12-12 21:23:18 (d2eee3ee2c280bed525caeef246401be5f68588c)](run.20251212212318.d2eee3ee2c280bed525caeef246401be5f68588c.html)
-- [2025-12-11 14:44:45 (12dfc2be1919aafff58bea5c59b8a698c0967663)](run.20251211144445.12dfc2be1919aafff58bea5c59b8a698c0967663.html)
+
+## Canonical Runs
+{% include canonical-list.md %}
```

Comment:
Import the list partial. Now we can simply prepend the list when a new summary visualiser report is generated rather than rebuilding the index page on a separate scheduled workflow.